Lib/test/test_pprint.py

```diff
@@ -285,11 +285,11 @@ def test_sorted_dict(self):
         self.assertEqual(pprint.pformat([d, d]),
             "[{'a': 1, 'b': 1, 'c': 1}, {'a': 1, 'b': 1, 'c': 1}]")
 
-        # The next one is kind of goofy.  The sorted order depends on the
+        # The next one is kind of odd.  The sorted order depends on the
         # alphabetic order of type names:  "int" < "str" < "tuple".  Before
         # Python 2.5, this was in the test_same_as_repr() test.  It's worth
         # keeping around for now because it's one of few tests of pprint
-        # against a crazy mix of types.
+        # against a random mix of types.
```

Comment:
From the technical side, I would prefer 'haphazard' or 'arbitrary', as 'random' implies a probability distribution.